collect_app/src/main/java/org/odk/collect/android/adapters/FileArrayAdapter.java

```diff
@@ -76,10 +82,16 @@ void onBind(DriveListItem item) {
                 Drawable d = ContextCompat.getDrawable(getContext(), R.drawable.form_state_blank);
                 imageView.setImageDrawable(d);
                 checkBox.setVisibility(View.VISIBLE);
+                if (isNewVersion(item)) {
+                    formUpdateAlert.setVisibility(View.VISIBLE);
+                } else {
+                    formUpdateAlert.setVisibility(View.GONE);
+                }
             } else {
                 Drawable d = ContextCompat.getDrawable(getContext(), R.drawable.ic_folder);
                 imageView.setImageDrawable(d);
                 checkBox.setVisibility(View.GONE);
+                formUpdateAlert.setVisibility(View.GONE);
```

Comment:
The view is gone by default but we need to set it here to avoid problem while scrolling when elements are reused am I right?

Reply:
yep, absolutely right!